grammars/csharp.cson

```diff
@@ -1824,7 +1824,7 @@ repository:
                   }
                 ]
               "7":
-                name: "variable.other.cs"
+                name: "entity.name.variable.local.cs"
```

Comment:
As far as I remember, we are currently not just keeping the grammar from upstream here, but also applying a few patches specific to GitHub C# code rendering engine. For example, this includes the following patch: https://github.com/atom/language-csharp/commit/e3a379f48f7a0db790af326046562584651f4d5e These `name` directives are used to determine which color to use for the highlighting. So we should test this grammar here https://github-lightshow.herokuapp.com/ to avoid regressions. E.g. in the grammar used by GitHub and Atom, we change a few `name` directives to enable the highlighting of method calls.